user/enterprise/platform-tips.md

```diff
@@ -393,6 +391,32 @@ $ curl -H "Authorization: token XXXXX" https://travis.example.com/api/uptime
 If everything is up and running, it answers with a `HTTP 200 OK`, or in case of failure with a `HTTP 500 Internal Server Error`.
 
 
+## Configuring Backups
+
+This section explains how you integrate Travis CI Enterprise in your backup strategy. Here, we'll talk about two topics:
+
+- The encryption key
+- The data directories
+
+### Encryption key
+
+Without the encryption key you cannot access the information in your production database. To make sure that you can always recover your database, make a backup of this key.
+
+> Without the encryption key the information in the database is not recoverable.
+
+To make a backup, please follow these steps:
+
+1. open a ssh connection to the platform machine
+2. run `travis bash`. This will open a bash session with `root` privileges into the Travis container.
```

Comment:
These often make more sense when you say what/why you're doing first, then how.

Reply:
Addressed in both your commit and my latest: https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4